R/drive_mv.R

```diff
@@ -37,20 +37,16 @@ build_drive_mv <-
       spf("Input `folder` must be a `gfile`. See `drive_file()`")
     }
 
-    url <- file.path(
-      .drive$base_url_files_v3,
-      paste0(
-        file$id,
-        "?addParents=",
-        folder$id,
-        "&removeParents=",
-        file$kitchen_sink$parents[[1]]
-      )
-    )
 
-    build_request(endpoint = url,
-                  token = token,
-                  method = "PATCH")
+    build_request(endpoint = paste0(
+      file$id,
+      "?addParents=",
+      folder$id,
+      "&removeParents=",
+      file$kitchen_sink$parents[[1]]
+    ),
```

Comment:
this is kind of a problem with my internal functions, but right now in order for `params` to get passed to the `query`, it must use the `GET` method, otherwise the `params` pass to the body.